### PR TITLE
Melvin update

### PR DESCRIPTION
**Pull request by britown88**
Melvin's all usable and everything now
@pconerly, please review

### CODE REVIEW

**Comment by pconerly** [comment]
Step 1:
`git checkout test/ master` 
and then commit the change to undelete my shit.

File: src/todoIssueGenerator.py [hunk 1]
```diff
@@ -131,9 +128,12 @@ def buildIssue(todo):
...
     ret = {}
-    
-    ret['title'] = renderTemplate(titleTemplate, data)
-    ret['body'] = buildIssueBody(data)
+   
+    try: 
+        ret['title'] = renderTemplate(titleTemplate, data)
+        ret['body'] = buildIssueBody(data)
+    except:
+        pass
```
Comment (pconerly) [inline src/todoIssueGenerator.py:136]: Do we need better exception handling here?

**Comment by pconerly** [comment]
I need to pull down this code to finish reviewing, I'll get to it tonight or tomorrow.

File: src/todoIssueGenerator.py (hunk 1 above)
Comment (britown88) [inline src/todoIssueGenerator.py:136]: in a word, yes... for some reason, some sort of weird data gets captured that causes exceptions....I'll see if I can nail down what's getting captured so I can throw them out before this is reached

**Comment by pconerly** [comment]
LGTM

**Pull request merged**